recipes/psiclass/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: psiclass
+  version: 1.0.2
+
+source:
+  url: https://github.com/splicebox/PsiCLASS/archive/v1.0.2.tar.gz
+  sha256: 3958e38e071d02c239fb8748f213fe96c724c06860fbc381cdeba5331bafa6ae
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - samtools 0.1.19.*
+    - zlib
+  run:
+    - zlib
```

Comment:
zlib can be removed from the run section

Reply:
I've removed the zlib from the file.